src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights/TelemetryPublisherApplicationInsights.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.ApplicationInsights;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.FeatureManagement.Telemetry.ApplicationInsights
+{
+    /// <summary>
+    /// Used to publish data from evaluation events to Application Insights
+    /// </summary>
+    public class TelemetryPublisherApplicationInsights : ITelemetryPublisher
```

Comment:
> TelemetryPublisherApplicationInsights

That name seems backwards. Like it should be `ApplicationInsightsTelemetryPublisher`